### DISCUSSION TITLE
Decimal rounding for Pareto Panel

### DISCUSSION THREAD

**Discussion by AkshayIIoT** [grafana/grafana#63064] [Dashboards, Panels & Visualizations]
Currently on Grafana v9.3.0

Please add decimal rounding function for Pareto Panel.

Currently when I display data on Pareto chart after performing math on the query results, Pareto panel does not have settings like other panels which allows us to add decimal rounding and the result shows like this:

![image](https://user-images.githubusercontent.com/124737022/217368036-d584a748-193b-450c-859e-c4345e6d019f.png)

Like Stat panel, if we could add decimal rounding, it would be very appealing

![image](https://user-images.githubusercontent.com/124737022/217368002-6b4981c6-3674-4078-9613-e7e3a6a0bbdc.png)

**Comment by gelicia** [comment]
Hello! Grafana does not maintain the Pareto panel. I would suggest logging this feature request here https://github.com/isaozler/pareto-chart
